django-tuttle/provider/models.py

```diff
@@ -3,13 +3,19 @@
 
 
 class Provider(models.Model):
-    name = models.CharField(null=True, max_length=50)
+    """
+    Provider model: stores provider's name
+    """
+    name = models.CharField(max_length=50)
 
     def __str__(self):
         return self.name
 
 
 class Repository(models.Model):
+    """
+    Repository model: stores the name of the repository and the user who own the repo
+    """
     name = models.CharField(max_length=100)
     user = models.ForeignKey(settings.AUTH_USER_MODEL)
```

Comment:
user field shoudn't be a foreign key to the internal user model
a repository has a owner which can be both an organization or github user